packages/notifications/src/styled/StyledIcon.tsx

```diff
@@ -0,0 +1,20 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import React, { Children } from 'react';
+import styled from 'styled-components';
+import { getColor } from '@zendeskgarden/react-theming';
+
+export const StyledIcon = styled(({ children, ...props }) =>
+  React.cloneElement(Children.only(children), props)
+)`
+  position: absolute;
+  right: ${props => props.theme.rtl && `${props.theme.space.base * 6}px`};
+  left: ${props => !props.theme.rtl && `${props.theme.space.base * 6}px`};
+  margin-top: ${props => props.theme.space.base / 2}px;
+  color: ${props => props.hue && getColor(props.hue, 600, props.theme)};
```

Comment:
What purpose is this `color` property serving?

Reply:
This `color` property is styling the icon for `<Notification>`. A hue is calculated using the type, e.g. `success`, `error`, etc from `<Notification>`, and is then passed down as a prop to `<StyledIcon>`.

There isn't a `hue` prop passed to `<StyledIcon>` from `<Alert>`, so the Icon in `<Alert>` gets its color inherited from its parent (which determines the font color based on the type).